core/src/main/kotlin/org/evomaster/core/search/service/Archive.kt

```diff
@@ -385,7 +385,13 @@ class Archive<T> where T : Individual {
             }
 
             val limit = apc.getArchiveTargetLimit()
-            if (current.size < limit) {
+
+            /**
+             individual can be added only if the target k is not covered.
+             If a target is covered and a 'better'(e.g., shorter) individual appears,
+             it would be handled as replacement, see line 404
+             */
```

Comment:
do not refer to a line by number, as that ll likely change at the next modification of this file :)

Reply:
modified :)